src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderSpanMultipleLines.java

```diff
@@ -0,0 +1,52 @@
+//non-compiled with javac: contains specially crafted set of imports for testing
```

Comment:
Can we make it compilable ?
You do not have fancy imports

Reply:
Done. Moved from `src/test/resources-noncopilable` to `src/test/resources`.